Source/NSBitmapImageRep+BMP.m

```diff
@@ -0,0 +1,444 @@
+/*
+   NSBitmapImageRep+PNG.m
+
+   Methods for loading .png images.
+
+   Copyright (C) 2003-2013 Free Software Foundation, Inc.
```

Comment:
Wrong dates

Reply:
Not wrong, because that is when the BMP file support was added. :)

Reply:
File removed.